go/internal/discordlog/discord_test.go

```diff
@@ -0,0 +1,16 @@
+package discordlog
+
+import (
+	"os"
+	"testing"
+)
+
+func TestShareQRLink(t *testing.T) {
+	if os.Getenv("BERTY_TEST_DISCORD") != "1" {
```

Comment:
We could replace this env variable by the one containing the QRCodeRoom (if empty -> t.Skip())